__tests__/car.test.js

```diff
@@ -0,0 +1,28 @@
+import Car from '../src/Car.js';
+
+describe('자동차 테스트', () => {
+  const CAR_NAME = '싼타페';
+  let car;
+
+  beforeEach(() => {
+    car = new Car({ name: CAR_NAME });
+  });
+
+  test('자동차는 이름을 가진다', () => {
+    expect(car.name).toBe(CAR_NAME);
+  });
+
+  test('초기 위치는 0이다', () => {
+    expect(car.position).toBe(0);
+  });
+
+  test('위치는 STEP_SIZE(1) 만큼 전진한다', () => {
+    expect(car.position).toBe(Car.INITIAL_POSITION);
+
+    car.moveForward();
+    expect(car.position).toBe(Car.INITIAL_POSITION + Car.STEP_SIZE * 1);
+
+    car.moveForward();
+    expect(car.position).toBe(Car.STEP_SIZE * 2);
+  });
```

Comment:
```suggestion
  test('위치는 STEP_SIZE(1) 만큼 전진한다', () => {
    expect(car.position).toBe(0);

    car.moveForward();
    expect(car.position).toBe(1);

    car.moveForward();
    expect(car.position).toBe(2);
  });
```

다만 상수를 통해 toBe의 값을 표현하는게 아니라 이렇게 직접적인 수치를 입력해야 테스트를 읽는 사람 입장에서 더 수월하지 않을까요!?

작성자가 아닌 사람들의 입장을 항상 고려해주시면 좋을 것 같아요!